public/app/plugins/datasource/tempo/traceql/TraceQLEditor.tsx

```diff
@@ -101,6 +102,12 @@ function setupActions(editor: monacoTypes.editor.IStandaloneCodeEditor, monaco:
   });
 }
 
+function setupRegisterInteractionCommand(editor: monacoTypes.editor.IStandaloneCodeEditor): string | null {
+  return editor.addCommand(0, function (_, label, type) {
+    reportInteraction('grafana_traces_traceql_completion', { datasourceType: 'tempo', label, type });
```

Comment:
I wonder do we need some sanitization here? It's probably a low chance there being some personal data considering that would be high cardinality but still, it may be worth checking if we have something in place somewhere in the pipeline or if this is non-issue.